src/app/widgets/JogControl/index.jsx

```diff
@@ -572,6 +585,45 @@ class AxesWidget extends PureComponent {
     }
 
     shuttleControlEvents = {
+        JOG_A_PLUS: { // Jog A+
+            id: 100,
+            title: 'Jog: A+',
+            keys: ['ctrl', '6'].join('+'),
+            cmd: 'JOG',
+            payload: {
+                axis: { [AXIS_A]: 1 },
+            },
+            preventDefault: false,
+            isActive: true,
+            category: JOGGING_CATEGORY,
+            callback: this.shuttleControlFunctions.JOG,
+        },
+        JOG_A_MINUS: { // Jog A-
+            id: 101,
+            title: 'Jog: A-',
+            keys: ['ctrl', '4'].join('+'),
+            cmd: 'JOG',
+            payload: {
+                axis: { [AXIS_A]: -1 },
+            },
+            preventDefault: false,
+            isActive: true,
+            category: JOGGING_CATEGORY,
+            callback: this.shuttleControlFunctions.JOG,
+        },
```

Comment:
The "cmd" property needs to be unique, that is why A- is not showing up on the list